nautobot/users/filters.py

```diff
@@ -40,6 +49,64 @@ class UserFilterSet(BaseFilterSet):
         to_field_name="name",
         label="Group (name)",
     )
+    has_changes = RelatedMembershipBooleanFilter(
+        field_name="changes",
+        label="Has Changes",
+    )
+    changes = NaturalKeyOrPKMultipleChoiceFilter(
+        to_field_name="user_name",
+        queryset=ObjectChange.objects.all(),
+        label="Object Changes (ID or user_name)",
+    )
+    has_logentry = RelatedMembershipBooleanFilter(
+        field_name="logentry_set",
+        label="Has Changes",
```

Comment:
Given that LogEntry is an admin-only model, and is potentially confused with ObjectChange, I don't think we necessarily should expose this FK to end users.